projects/js-packages/licensing/components/activation-screen-success-info/product-details/jetpack-backup-daily.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+/**
+ * The Jetpack Backup Daily Product Details component.
+ *
+ * @param {object} props -- The properties.
+ * @param {number} props.dashboardUrl -- The url that links to the site dashboard
+ * @returns {React.Component} The `JetpackBackupDailyProductDetails` component.
+ */
+const JetpackBackupDailyProductDetails = props => {
+	const { dashboardUrl } = props;
+	return (
+		<div>
+			<h1>
+				{ __( 'Your Jetpack Daily Backup is active!', 'jetpack' ) }{ ' ' }
+				{ String.fromCodePoint( 0x1f389 ) }
+				{ /* Celebration emoji 🎉 */ }
```

Comment:
Do you think we could add the space and the emoji in the translatable string with a `sprintf`, so things are easier to translate?